app/src/components/HealthContainer.tsx

```diff
@@ -10,8 +10,8 @@ interface ContainerProps {
 const HealthContainer: React.FC<ContainerProps> = ({ name, value, health }) => {
     const bg: string = statusColour(health);
     return (
-        <div className="flex justify-center text-black">
-            <div className={`rounded-lg shadow-xl font-bold text-lg responsive-width m-4 bg-${bg}-550`}>
+        <div className="flex w-full text-black">
+            <div className={`rounded-lg shadow-xl font-bold text-lg w-full flex-grow bg-${bg}-550`}>
```

Comment:
This change messes up the formatting for the sensors and individual sensor pages. Is there a reason this was changed?